cms/core/viewsets.py

```diff
@@ -47,3 +49,53 @@ class ContactDetailsViewSet(SnippetViewSet):
 
     index_view_class = ContactDetailsIndex
     chooser_viewset_class = ContactDetailsChooserViewset
+
+
+class GlossaryTermCreateView(SnippetCreateView):
+    def save_instance(self) -> GlossaryTerm:
+        """Automatically set the last updating user on Glossary Term on creation and modification."""
+        instance: GlossaryTerm = super().save_instance()
+        instance.updated_by = self.request.user
+        instance.save(update_fields=["updated_by"])
+        return instance
+
+
+class GlossaryTermEditView(SnippetEditView):
+    def save_instance(self) -> GlossaryTerm:
+        """Automatically set the last updating user on Glossary Term on creation and modification."""
+        instance: GlossaryTerm = super().save_instance()
+        instance.updated_by = self.request.user
+        instance.save(update_fields=["updated_by"])
+        return instance
+
+
+class GlossaryTermsIndex(SnippetIndexView):
+    list_display: ClassVar[list[str | Column]] = [
+        "title",
+        UpdatedAtColumn(),
+        UserColumn("updated_by"),
+    ]
+
+
+class GlossaryChooseView(SnippetChooseView): ...
+
+
+class GlossaryChooseResultsView(SnippetChooseResultsView): ...
```

Comment:
it would be good to include the last updated and updated by columns to both the choose and choose results views

Reply:
[Added owner field + updated the snippet views](https://github.com/ONSdigital/dis-wagtail/pull/110/commits/a800f0fc6c8661a7c16de3cbc9f0053d01c026a5)